homeassistant/components/zwave/__init__.py

```diff
@@ -310,6 +317,17 @@ def log_all(signal, value=None):
 
         dispatcher.connect(log_all, weak=False)
 
+    @websocket_api.async_response
+    async def websocket_network_status(hass, connection, msg):
+        message = websocket_api.result_message(
+            msg['id'], [network.state, network.state_str]
```

Comment:
Are fetching these properties doing I/O ?

Reply:
It's coming from python-openzwave, so it's possible. I wasn't able to find docs on python-openzwave for those properties, but they're used around line 679 which gets run through hass.add_job()

Reply:
that's inside a sync method. You need to look at Open Z-Wave C code to see if it's doing I/O.

Also, don't return a list, return a dictionary or else we can't extend it in the future.